vpr/src/place/place_util.h

```diff
@@ -205,5 +213,42 @@ int get_initial_move_lim(const t_placer_opts& placer_opts, const t_annealing_sch
 ///@brief Returns the standard deviation of data set x.
 double get_std_dev(int n, double sum_x_squared, double av_x);
 
+//Placement checkpoint
+/**
+ * @brief Data structure that stores the placement state and saves it as a checkpoint.
+ *
+ * The placement checkpoints are very useful to solve the problem of critical 
+ * delay oscillations, expecially very late in the annealer. 
+ *
+ *   @param cost The weighted average of the wiring cost and the timing cost.
```

Comment:
Should also comment sTNS and sWNS, and I think the cost is in fact a full cost structure of all the placer cost components.

Reply:
Agree. That is even not used in the code. Fixed by removing them from the class definition.